drivers/clk/bcm/clk-bcm2835.c

```diff
@@ -523,8 +523,13 @@ static long bcm2835_pll_rate_from_divisors(unsigned long parent_rate,
 static long bcm2835_pll_round_rate(struct clk_hw *hw, unsigned long rate,
 				   unsigned long *parent_rate)
 {
+	struct bcm2835_pll *pll = container_of(hw, struct bcm2835_pll, hw);
+	const struct bcm2835_pll_data *data = pll->data;
 	u32 ndiv, fdiv;
 
+	rate = max(data->min_rate, rate);
+	rate = min(data->max_rate, rate);
+
```

Comment:
I was off on the user qpu branch and just now got a chance to fix this.

Note that github now allows maintainers to edit the contents of pull requests, which is a nice way of dealing with style nitpicks instead of round-tripping to the submitter.